index.html

```diff
@@ -189,10 +189,46 @@ <h2 class="about-primary-text">About me</h2>
             </ul>
           </div>
         </ul>
-      </div>
+      </div>  
 
     </div>
   </div>
 </section>
+
+
+
+
+
+<section class="contact">
+  <div class="bottom">
+<p class="text">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</p>
+<form action="https://formspree.io/f/mlevodye" method="POST" >
+<input type="text" id="name" name="user_name" maxlength="30" placeholder="Full name" required>
+<input type="email" id="mail" name="user_email" placeholder="Email address" required>
+<textarea type="text" name="message" placeholder="Enter text here" maxlength="500" required></textarea>
+<button type="submit" value="submit">Get in touch</button>
+</form>
+  </div>
+</section>
```

Comment:
- [ ] Good job styling your portfolio contact form section. However, according to the project requirements, you are required to use the CSS flexbox to align the items of this section. Kindly do so.

![image](https://user-images.githubusercontent.com/86318284/199240391-cbd2d33f-e535-4c4a-981a-c5ffc549d95d.png)